ansible/extension/setup/initial.yml

```diff
@@ -0,0 +1,16 @@
+hosts: all
+become: yes
+tasks:
+  - name: create the 'ubuntu' user
+    user: name={USERNAME} append=yes state=present createhome=yes shell=/bin/bash
```

Comment:
Hi @NoahNsimbe :
Could you shed some light on how the {username} field affects the readiness of the VMs in the cluster?

Reply:
Hello @Baalmart , 
The {username} field does not have any effect on the readiness of the VMs in the cluster, It is only used during setup (installing dependencies) since it's a non-root/ unprivileged user.

Reply:
> Hello @Baalmart,
> The {username} field does not have any effect on the readiness of the VMs in the cluster, It is only used during setup (installing dependencies)

Great, I thought so too. :)